src/container/core.jl

```diff
@@ -244,3 +234,6 @@ function send_message(
     )
 end
 
+function getindex(container::Container, index::String)
```

Comment:
Similar comment to above. I see how this is very useful. I just wonder if `container[1]` returning an agent is the correct way to do it.